src/index.css

```diff
@@ -6,6 +12,8 @@
   border-radius: 4px;
   box-shadow: 0 0 60px 0 var(--black-5);
   background-color: #fafafa;
+  font-size: 20px;
+  text-decoration: none;
 }
 
```

Comment:
- [ ] Kindly, stick to the design, as you can use in the original design we don't have `Bookstore-CMS` underlined and space with book menu

![rr26](https://user-images.githubusercontent.com/55635977/135471927-c4326e1e-c080-4132-ac44-79a14b8d802e.png)

Reply:
Hello, Emmy please note that my name is not black-box. I have implemented the requested changes